spec/controllers/requested_accounts_campaigns_controller_spec.rb

```diff
@@ -49,21 +49,25 @@
         expect(RequestedAccount.count).to eq(1)
       end
 
-      it 'creates the accounts if user is authorized' do
-        stub_account_creation
-        allow(controller).to receive(:current_user).and_return(admin)
-        allow(UserImporter).to receive(:new_from_username).and_return(user)
-        post :create_accounts, params: { campaign_slug: campaign.slug }
-        expect(response.status).to eq(200)
-        expect(RequestedAccount.count).to eq(0)
+      if Features.enable_account_requests?
```

Comment:
Same thing here; use `allow` to turn the feature on for these tests, rather than skip the tests.